scripts/components/ModelViewer/ModelViewer.js

```diff
@@ -14,7 +14,6 @@ const ModelViewer = (props) => {
       onClick={handleClick}
       style={{ width: '100%', height: '100%' }}
       src={modelPath}
-      alt='A 3D model of an astronaut'
```

Comment:
Still no option to set an alternative text for authors, see general comments on a11y.